dashboard/scripts/api-client.js

```diff
@@ -0,0 +1,70 @@
+//Enable strict mode for ES6 support
+'use strict';
+
+const baseUrl = 'https://sjtek.nl/api/';
+
+(function setInitialData(){
+  httpRequest("GET", "https://sjtek.nl/api", function(data) {
+
+    data = JSON.parse(data.responseText);
```

Comment:
Setting data should be a separate method (for reusing when using websocket updates)